src/main/server/enable_ssl.sh

```diff
@@ -157,11 +157,24 @@ if [ "$chainFileExist" != "$privateKeyFileExist" ]; then
 fi
 
 get_freedomain(){
+
+  check_marketplace() {
+    if curl -s http://169.254.169.254/latest/meta-data/ >/dev/null; then
```

Comment:
When I run this command in OVH instance, it returns some content. It means then statement works. 

Just for your reference, when I run following command in OVH instance, it prints out "hello". I think it should not print out "hello"
```
if curl -s http://169.254.169.254/latest/meta-data/ >/dev/null; then echo "hello"; fi;
```
```
ubuntu@ci:~$  if curl -s http://169.254.169.254/latest/meta-data/ >/dev/null; then echo "hello"; fi;
hello
```


When I test it on my local computer, it just stucks 
```
if curl -s http://169.254.169.254/latest/meta-data/ >/dev/null; then echo "hello"; fi;
```